### PR TITLE
Gold Rings Price Nerf + Craftable Rings (Gold/Steel)

### PR DESCRIPTION
**Pull request by Sarkness**
## About The Pull Request

This pull request is after many complaints about the gold rings making the Economy crashes after players found you you can sell gold, buy gold and repeat the process (mainly with Steward, a smith and merchant)
This also adds the Emerald Ring that may be craftable if players are lucky enough to find an emerald gem (named Gemerald) and a gold ingot

## Why It's Good For The Game

Makes economy somewhat more stable + Adds the possibility to have Portable SCOMs more open to everyone (besides having it only for King and Heartfelt)

### CODE REVIEW

**Comment by OrbitalParadise** [comment]
I think emerald rings should be considered a magic item, the lack of communication can be solved in the future by use of magic or an artificer. If everyone has emerald rings it's going to shit up everything

**Comment by NGGJamie** [comment]
Agree with the poster above, it would be too much if everyone was running around with SCOM rings. I would be completely down for having craftable emerald rings that don't have the SCOM abilities, though.

**Comment by Sarkness** [comment]
Fixed, Rings are now non-SCOM Rings

File: code/modules/clothing/rogueclothes/rings.dm [hunk 1]
```diff
@@ -20,7 +20,7 @@
 /obj/item/clothing/ring/gold
 	name = "gold ring"
 	icon_state = "ring_g"
-	sellprice = 70
+	sellprice = 47
```
Comment (BadAtThisGame302) [inline code/modules/clothing/rogueclothes/rings.dm:23]: Should be 50 or 45, very weird to leave it on a 7.

Comment (Sarkness) [reply code/modules/clothing/rogueclothes/rings.dm:23]: I mean, it's more to round it to 141

Comment (UbaserB) [reply code/modules/clothing/rogueclothes/rings.dm:23]: > I mean, it's more to round it to 141

that is also equally weird

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm [hunk 2]
```diff
@@ -45,3 +45,46 @@
...
+
+/datum/anvil_recipe/valuables/topazg
+	name = "Gold Topaz Ring (+ Topaz)"
+	req_bar = /obj/item/ingot/gold
+	additional_items = list(/obj/item/roguegem/yellow)	
+	created_item = /obj/item/clothing/ring/topaz
+
+/datum/anvil_recipe/valuables/emerings
+	name = "Silver Emerald Ring (+ Gemerald)"
+	req_bar = /obj/item/ingot/steel
```
Comment (JTGSZ) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm:69]: steel ingots for creation of silver rings, unless these are supposed to be steel rings in which case you change the name.

**Comment by Sarkness** [comment]
Unfortunately there's no silvers yet, so if needed you can just remove it I
guess.
Otherwise until silver is really added I suppose?

**Pull request merged**